tasks/dataverse-install.yml

```diff
@@ -93,22 +90,60 @@
     path: /tmp/dvinstall
     owner: '{{ dataverse.glassfish.user }}'
 
+- name: make backup copy of install.py, to know whether it changed and we need to patch it later
+  copy:
+    src: /tmp/dvinstall/install.py
+    dest: /tmp/dvinstall/install.py.prev
+    remote_src: true
+  register: install_py # this will be marked changed if there was no install.py.prev and thus no previous install, or the downloaded is different from the previous
+
+- name: copy install.py to install_ansible.py
+  copy:
+    src: /tmp/dvinstall/install.py
+    dest: /tmp/dvinstall/install_ansible.py
+    remote_src: true
+  when: install_py.changed
+
+- name: patch install_ansible.py so that it does not ask for password
+  lineinfile:
+    path: /tmp/dvinstall/install_ansible.py
+    regexp: '^gfAdminLoginStatus'
+    state: absent
+
+- name: patch install_ansible.py for available jdbc driver version
+  lineinfile:
+    path: /tmp/dvinstall/install_ansible.py
+    regexp: '^pgJdbcDriver '
+    line: 'pgJdbcDriver = "postgresql-42.2.9.jar"'
+
+- name: patch install_ansible.py so that it allows for java 11 also
```

Comment:
I think this would better be incorporated into https://github.com/IQSS/dataverse/blob/develop/scripts/installer/install.py

Reply:
I think I will remove this lineinfile. It really depends what version of dataverse are you installing. Maybe the better solution is to download the jar requested by the newer installer.

Reply:
I fixed this in cee8327

Reply:
the jdbcdriver version is already a configurable group_var, but we're modifying install.py downstream rather than contributing changes to the primary repo...

Reply:
Well, is it likely that you will accept java11 support in install.py in the dataverse repository?

Reply:
@pallinger hi! Thanks for all these pull requests! If Java 11 support is something you want the upstream install.py script to support, please open an issue at https://github.com/IQSS/dataverse/issues . Thanks!

Reply:
@pallinger I played with install.py a bit today. will this take care of later java versions and the password prompt?
https://github.com/IQSS/dataverse/compare/develop...OdumInstitute:6982_install_py_java_check